services/queue/schemas/v1/task-group-response.yml

```diff
@@ -0,0 +1,31 @@
+$schema: "/schemas/common/metaschema.json#"
+title:              "Task-Group Response"
+description: |
+  Response containing information about a task group.
+type:               object
+properties:
+  taskGroupId:
+    title:          "Task-Group Identifier"
+    description: |
+      Identifier for the task-group being sealed.
+    type:           string
+    pattern:        {$const: slugid-pattern}
+  sealed:
+    type:           string
+    format:         date-time
+    title:          "Sealing timestamp"
+    description: |
+      Date and time when task group was sealed.
+  expires:
+    type:           string
+    format:         date-time
+    title:          "Task group expiry date"
+    description: |
+      Date and time when task group expires.
```

Comment:
It will be worth noting here that this date is after the last expiration of any task in the task group, and for an unsealed taskgroup could change to a later date.